integration/yarn/src/test/java/alluxio/yarn/ContainerAllocatorTest.java

```diff
@@ -186,7 +186,10 @@ public Void answer(InvocationOnMock invocation) throws Throwable {
       }
     };
   }
-
+  /*
+   * Counts the hosts of containers and check if the number of hosts exceeds the maximum hosts
```

Comment:
"Verifies that no more than maxContainersPerHost of the containers are located on the same host."